docker/docker-compose-dev.yml

```
@@ -0,0 +1,124 @@
+version: '3.2'
+services:
+
+  # ICUBAM service
+  app-server:
+    build:
+      context: "${ICUBAM_COMPOSE_CONTEXT:-..}"
```

See comment below about the mount
```suggestion
      context: "${ICUBAM_COMPOSE_CONTEXT:-.}"
```